tests/functional/firefox/test_android.py

```diff
@@ -15,6 +16,30 @@ def test_family_navigation(base_url, selenium):
     assert page.family_navigation.is_menu_displayed
 
 
+@pytest.mark.flaky(reruns=1)
```

Comment:
As it submits to basket I'm making the assumption we might get the same issue as we see on other newsletters.